lib/DatabaseLayer.py

```diff
@@ -9,8 +9,8 @@
 
 # imports
 import ast
-import sqlite3
 import pymongo
+#from pymongo import UpdateOne
```

Comment:
It'd be prefereable to remove all commented out code, it is not executed either way.